src/main/java/seedu/algobase/logic/parser/ParserUtil.java

```diff
@@ -260,4 +265,52 @@ public static LocalDateTime parseDate(String date) throws ParseException {
         }
     }
 
+    /** Parses a {@code String model enum} into an {@code ModelEnum}.
+     *
+     * @throws ParseException if the given {@code string model enum} is invalid.
+     */
+    public static TabEnum parseTabType(String tabType) throws ParseException {
+        try {
+            return TabEnum.valueOf(tabType.toUpperCase());
+        } catch (IllegalArgumentException e) {
+            throw new ParseException(e.toString());
+        }
+    }
+
+    /** Parses a {@code String model enum} into an {@code ModelEnum}.
+     *
+     * @throws ParseException if the given {@code string model enum} is invalid.
+     */
+    public static Index parseTabIndex(String tabIndex) throws ParseException {
+        try {
+            return ParserUtil.parseIndex(tabIndex);
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, SwitchTabCommand.MESSAGE_USAGE), pe);
+        }
+    }
+
+
+    /** Parses a {@code String model enum} into an {@code ModelEnum}.
+     *
+     * @throws ParseException if the given {@code string model enum} is invalid.
+     */
+    public static ModelEnum parseModelTab(String modelTab) throws ParseException {
+        try {
+            return ModelEnum.valueOf(modelTab.toUpperCase());
+        } catch (IllegalArgumentException e) {
+            throw new ParseException(e.toString());
+        }
+    }
+
+    /** Parses a {@code String model index} into an {@code ModelEnum}.
```

Comment:
Maybe you should write `modelIndex` in the JavaDoc as well - just to keep it consistent with the method signature.